fe/fe-core/src/main/cup/sql_parser.cup

```diff
@@ -1827,6 +1837,10 @@ drop_stmt ::=
     {:
         RESULT = new DropResourceStmt(resourceName);
     :}
+    | KW_DROP KW_SQL_BLOCK_RULE ident_list:ruleNames
```

Comment:
Can different user have same rule name? If yes, how to handle it? If no, how to check it?
Maybe you should write like: `drop sql_block_rule FROM username ...`

Reply:
The current design is that rule names are not allowed to be repeated, so I think it's easier to use? Maybe we can support configuring multiple users